packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -162,6 +210,22 @@ export default class NodeResolver {
         moduleName: parts[0],
         subPath: parts[1],
       };
+
+      let alternativeModules = await this.findAlternativeNodeModules(
+        resolved.moduleName,
+        dir,
+      );
+
+      if (alternativeModules.length) {
+        throw new ThrowableDiagnostic({
```

Comment:
Not entirely sure how we'd make this work together with auto-install?

Maybe we should just get rid of auto-install?